src/components/CodeView/index.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import Refractor from 'react-refractor/all';
+
+import styles from './styles.module.scss';
+import './prism.css';
+
+type PublicProps = {
+  mimeType: string;
+  content: string;
+};
+
+export const getLanguage = (mimeType: string) => {
+  switch (mimeType) {
+    case 'application/javascript':
+    case 'text/javascript':
+      return 'js';
+    case 'application/json':
+      return 'json';
+    case 'application/xml':
+      return 'xml';
+    case 'text/css':
+      return 'css';
+    case 'text/html':
+      return 'html';
+    default:
+      return null;
+  }
+};
+
+export const getLines = (content: string) => {
+  return content.replace(/\n$/, '').split('\n');
+};
+
+const CodeViewBase = ({ mimeType, content }: PublicProps) => {
+  const language = getLanguage(mimeType);
+
+  return (
+    <div className={styles.CodeView}>
+      <pre>
+        <code className={styles.lineNumbers}>
+          {getLines(content).map((_, i) => {
+            const lineNumber = i + 1;
+
+            return (
+              <span key={`line-number-${lineNumber}`}>{`${lineNumber}`}</span>
```

Comment:
I'm not totally sure how a separate column of line numbers will let us do what we want for review forms but if you already have a prototype then I believe you 😎

Reply:
Ok so that's what I wanted to double-check today and you are right, that won't be easy. I switched to a better version and I have a proof that it works as intended now:

![screen shot 2019-02-27 at 11 15 06](https://user-images.githubusercontent.com/217628/53483080-f90b4100-3a80-11e9-9cd8-bfc29517d2f8.png)